kore/test/Test/Kore/Rewrite/MockSymbols.hs

```diff
@@ -2325,7 +2331,7 @@ generatorSetup =
     ConsistentKore.Setup
         { allSymbols = filter doesNotHaveArguments symbols
         , allAliases = []
-        , allSorts = map fst sortAttributesMapping
+        , allSorts = filter (/= otherTopSort) $ map fst sortAttributesMapping
```

Comment:
Note that we still have this `otherTopSort` around in the tests. 
https://github.com/runtimeverification/haskell-backend/blob/ed3edb3e59eafbe0992b8d8aec458cb79773067b/kore/test/Test/Kore/Rewrite/MockSymbols.hs#L2112..L2128 appears a bit strange to me, not well-ordered at least.

Reply:
I like `overTheTopSort` :D